### PR TITLE
Add prometheus mongodb exporter.

### PR DESCRIPTION
**Pull request by dlorenc**
### Image Size

- [X] The Image is smaller in size than its common public counterpart.
- [ ] The Image is larger in size than its common public counterpart (please explain in the notes).

Notes:

### Image Vulnerabilities

- [X] The Grype vulnerability scan returned 0 CVE(s).

Notes:

### Basic Testing - K8s cluster

- [X] The container image was successfully loaded into a kind cluster.

Notes:

### Basic Testing - Package/Application

- [X] The application is accessible to the user/cluster/etc. after start-up.

Notes:

### Helm

- [X] A Helm chart has been provided and the container image can be used with the chart.  If needed, please add a -compat package to close any gaps with the public helm chart.

Notes:

### Processor Architectures

- [X] The image was built and tested for x86_64.
- [X] The image was built and tested for aarch64.

Notes:

### Functional Testing + Documentation

- [X] Functional tests have been included and the tests are passing.  All tests have been documnted in the notes section.

Notes:

### Environment Testing + Documentation

- [X] There has not been a request and/or there is no indication that this image needs tested on a public cloud provider.

Notes:

### Version

- [X] The package version is the latest version of the package.  The latest tag points to this version.

Notes:

### Dev Tag Availability

- [X] There is a dev tag available that includes a shell and apk tools (by depending on 'wolfi-base')

Notes:

### Access Control + Authentication

- [X] The image runs as `nonroot` and GID/UID are set to 65532 or upstream default

### ENTRYPOINT

- [X] applications/servers/utilities set to call main program with no arguments e.g. [redis-server]

### CMD

- [X] For server applications give arguments to start in daemon mode (may be empty)

### Environment Variables

- [ ] Environment variables added.

### SIGTERM

- [X] The image responds to SIGTERM (e.g., `docker kill $(docker run -d --rm cgr.dev/chainguard/nginx)`)

### Logs

- [X] Error logs write to stderr and normal logs to stdout. Logs DO NOT write to file.

### Documentation - README

- [X] A README file has been provided and it follows the README template.

### CODE REVIEW

File: images/prometheus/tests/deploy.yaml
Comment (mattmoor) [inline images/prometheus/tests/deploy.yaml:1]: did you mean to add this file?

**Review by mattmoor — APPROVED** [review]
LGTM other than the extra file

File: images/prometheus/tests/deploy.yaml
Comment (dlorenc) [reply images/prometheus/tests/deploy.yaml:1]: Doh.

**Comment by dlorenc** [comment]
> LGTM other than the extra file

RFAL

**Review by mattmoor — APPROVED** [review]

**Pull request merged**